zion/src/test/scala/edu/uci/ics/cloudberry/zion/common/ConfigTest.scala

```diff
@@ -11,6 +11,17 @@ class ConfigTest extends Specification {
       Config.parseTimePair("43 minute") must_==(43 minutes)
       Config.parseTimePair("84 minutes") must_==(84 minutes)
     }
-
+    "parseFrameLengthLimit" in {
+      Config.parseFrameLengthLimit("20 B") must_==(8 * MemorySize.MB)
+      Config.parseFrameLengthLimit("0 KB") must_==(8 * MemorySize.MB)
+      Config.parseFrameLengthLimit("0.9 KB") must_==(8 * MemorySize.MB)
+      Config.parseFrameLengthLimit("1 KB") must_==(1 * MemorySize.KB)
```

Comment:
that's a weird setting, right? 
why I get `1K` when we give `1K`, but `8M` when we give `0.5KB`?